docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-feature-integration.md

```diff
@@ -1359,16 +1383,6 @@ Enable Javascript and CSS changes:
 console frontend:yves:build
 ```
 
-{% info_block warningBox "Verification" %}
```

Comment:
Why it was removed? Verification steps should stay. If they are not correct, please provide a better one.

Reply:
It can't be checked before other integrations. This verification is already mentioned in "marketplace product" and "Marketplace Product Offer + Cart".

Reply:
1. > Verification steps should stay. If they are not correct, please provide a better one.
2. It can, SoldByMerchantWidget is shown on PDP (Product detail page)